udms/angular.json

```diff
@@ -44,7 +44,14 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ],
-              "optimization": true,
+	      "optimization": {
```

Comment:
indent seems incongruous with rest of file

Reply:
vimrc was configured in a weird way which messed up indents. Corrected.